### PR TITLE
Add template for react-inertia

### PR DESCRIPTION
**Pull request by markstory**
I updated the installation docs a tiny bit. I chose not to include bootstrap with inertia-react as it didn't seem necessary.

### CODE REVIEW

File: stubs/inertia-react/webpack.mix.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+const mix = require("laravel-mix");
+
+mix.setPublicPath("./webroot")
```
Comment (ishanvyas22) [inline stubs/inertia-react/webpack.mix.js:3]: Let's be consistent and use single quotes here in whole file as I've used single quotes in [`webpack.mix.js` file](https://github.com/ishanvyas22/asset-mix/blob/master/stubs/inertia-vue/webpack.mix.js). After this change the test will pass and we can able to merge this in.

```suggestion
mix.setPublicPath('./webroot')
```

File: stubs/inertia-react/assets/js/Pages/Pages/Greet.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React from "react";
```
Comment (ishanvyas22) [inline stubs/inertia-react/assets/js/Pages/Pages/Greet.js:1]:
```suggestion
import React from 'react';
```

**Review by ishanvyas22 — CHANGES_REQUESTED** [review]
First of all thanks for the PR. I've requested few changes, it will be ready to merge after points is resolved.

File: stubs/inertia-react/webpack.mix.js (hunk 1 above)
Comment (markstory) [reply stubs/inertia-react/webpack.mix.js:3]: Sorry about that my local prettier setup was acting up.

**Review by ishanvyas22 — APPROVED** [review]

**Comment by ishanvyas22** [comment]
Thanks!

**Comment by ishanvyas22** [comment]
Released in [1.3.0](https://github.com/ishanvyas22/asset-mix/releases/tag/1.3.0).

**Pull request merged**